lib/beaker-puppet/wrappers.rb

```diff
@@ -45,7 +45,7 @@ def puppet(*args)
         options[:cmdexe] = true
         # we assume that an invocation with `puppet()` will have it's first argument
         # a face or sub command
-        if @options[:type] == 'git'
+        if @options && @options[:type] == 'git'
```

Comment:
We can drop this commit if we don't install the runtime package on the master node